authz/authz-tool/tool/src/webapp/vm/helper/chef_permissions.vm

```diff
@@ -23,13 +23,12 @@
 		<div class="viewNav" id="groupSelector">
 			<form name="viewForm" class="inlineForm" method="post" action="#toolForm($action)">
 				<input type="hidden" name="eventSubmit_doView_permission_option" value="view" />
-				<div class="">
+				<div class="spinnerBesideContainer">
 					<label for="groupSelection">
 					$thelp.getString("per.lis.selectgrp")
 					</label>
-					<select id="authzGroupSelection" name="authzGroupSelection" onchange="AUTHZ.disableControls();AUTHZ.showSpinner( 'viewSpinner' );document.viewForm.submit();">
-						<option value ="$siteRef" #if ($!viewRealmIds.contains($!siteId)) selected = "selected" #end>$thelp.getString("per.lis.site")</option>
-
+					<select id="authzGroupSelection" name="authzGroupSelection" onchange="SPNR.insertSpinnerAfter( this, null, null );document.viewForm.submit();">
+						<option value ="$siteRef" #if ($!viewRealmId.equals($!siteId)) selected = "selected" #end>$thelp.getString("per.lis.site")</option>
```

Comment:
What is put into the context was not changed by this PR. If `viewRealmId` was once in the context and is now not present, I believe this would be a regression caused by another ticket.